packages/devtools_app/lib/src/debugger/debugger_model.dart

```diff
@@ -695,13 +696,14 @@ List<Variable> _createVariablesForFields(
 
 // TODO(jacobr): gracefully handle cases where the isolate has closed and
 // InstanceRef objects have become sentinels.
-// TODO(jacobr): consider a new class name. This class is more just the data
-// model for a tree of Dart objects with properties rather than a "Variable".
-// TODO(elliette): Refactor to make name, ref, text named (and potentially
-// required?) parameters. Give ref a type.
-class Variable extends TreeNode<Variable> {
-  Variable._(this.name, ref, this.text, {int offset, int childCount})
-      : _ref = ref,
+class DartObjectNode extends TreeNode<DartObjectNode> {
+  DartObjectNode._({
+    this.name,
+    this.text,
+    GenericInstanceRef ref,
```

Comment:
make these required

Reply:
Looks like none of them can be required (for example, `DartObjectNode.text` only provides the `text` parameter).